### DISCUSSION TITLE
Feature Request: Multi GPU Support

### DISCUSSION THREAD

**Discussion by PenTeZtZMicZ** [hacksider/Deep-Live-Cam#732] [Ideas]
Hi devs, is it possible to add support for multiple GPU's. This will allow us to utilise our processing power for a smoother performance.

**Comment by hacksider** [comment]
It is but since I don't have any to test, we have to wait for someone that have to push the code. ☺️

**Reply by PenTeZtZMicZ** [reply]
> It is but since I don't have any to test, we have to wait for someone that have to push the code. ☺️

Is it somehow possible to allow us to test? Got 4 3090s to use

**Comment by ghengx** [comment]
Any updates for this?

**Reply by PenTeZtZMicZ** [reply]
> Any updates for this?

Nothing on my end. Haven't heard or seen anything since

**Comment by hacksider** [comment]
Will close this once we release 2.0 soon
